app/src/main/java/com/superproductivity/superproductivity/FullscreenActivity.java

```diff
@@ -52,7 +52,7 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
         frameLayout.addView(wv);
 
         // init JS here, as it needs an activity to work
-        jsi = new JavaScriptInterface(this, wv);
+        jsi = new CommonJavaScriptInterface(this, wv);
```

Comment:
This is not correct, since this effectively ignores the adjustments being made for different build flavors for google play and fdroid.

Reply:
The old code caused a compilation error on my machine, please feel free to revert it if it works on your computer.